### ISSUE TITLE
[c++2py] Default arguments in function wrapping not properly picked up

### ISSUE THREAD

**Issue by Wentzell** [TRIQS/triqs#469]
With TRIQS/unstable, when I  wrap the simple function
```c++
/// Some useless function
void test(int j, int k = 0, int l = 1){}
```

the default arguments are not properly picked up by c++2py. The desc file reads
```python
# Generated automatically using the command :
# c++2py.py test.hpp -p -m test -o test
from wrap_generator import *

# The module
module = module_(full_name = "test", doc = "", app_name = "test")

# All the triqs C++/Python modules

# Add here all includes beyond what is automatically included by the triqs modules
module.add_include("test.hpp")

# Add here anything to add in the C++ code at the start, e.g. namespace using
module.add_preamble("""
""")
module.add_function ("void test (int j, int k, int l)", doc = """Some useless function""")

module.generate_code()
```

**Comment by parcollet** [comment]
Most likely fixed in dev (cpp2py_alias) branch. To be checked.

**Comment by parcollet** [comment]
Solved in 1.6-dev
